src/main/java/com/wojciechbarwinski/demo/epic_board_games_shop/validations/OrderStatusChangeValidation.java

```diff
@@ -32,6 +34,7 @@ public void assertOrderStatusTransitionIsAllowed(OrderStatus currentStatus, Orde
         List<OrderStatus> orderStatuses = allStatuses.get(currentStatus);
 
         if (!orderStatuses.contains(nextStatus)) {
+            log.warn("Unauthorized attempt to change order status: changing from {} to {} is not allowed.", currentStatus, nextStatus);
```

Comment:
Forbidden zamiast unauthorized - to drugie sugeruje, że ktoś z innymi uprawnieniami mógłby to zrobić